### ISSUE TITLE
Wagon push may still not be updating CSS

### ISSUE THREAD

**Issue by jasongonzales23** [locomotivecms/wagon#43]
I am cautious to bring this issue up again. If for no other reason than I hate to be annoying.

I am seeing the same behavior, but oddly it is intermittent.

Under some conditions that I cannot seem to isolate, wagon push will properly skip assets that have not been updated. In this case it never reliably updates CSS file(s). In some other cases it reports that it is updating everything and CSS is still not updated. 

In short, I cannot use wagon to update CSS changes. Is it because I am compiling CSS with an outside program and not Compass? Can someone else verify this?

Here is my situation: I host Locomotive locally with v2.0.3 and am using Wagon 1.0.2.

I write SCSS and use Codekit to compile it to the public/stylesheets directory.

In all cases I am always issuing the push command with the "force" flag:

wagon push development -f
